packit/copr_helper.py

```diff
@@ -134,6 +110,95 @@ def create_copr_project_if_not_exists(
                 additional_packages=additional_packages,
                 additional_repos=additional_repos,
             )
+            return
+
+        delete_after_days = -1 if preserve_project else 60
+
+        fields_to_change = self.get_fields_to_change(
+            copr_proj=copr_proj,
+            additional_repos=additional_repos,
+            chroots=chroots,
+            description=description,
+            instructions=instructions,
+            list_on_homepage=list_on_homepage,
+            delete_after_days=delete_after_days,
+        )
+
+        if fields_to_change:
+            logger.info(f"Updating copr project '{owner}/{project}'")
+            for field, (old, new) in fields_to_change.items():
+                logger.debug(f"{field}: {old} -> {new}")
+
+            try:
+                self.copr_client.project_proxy.edit(
+                    ownername=owner,
+                    projectname=project,
+                    chroots=chroots,
+                    description=description,
+                    instructions=instructions,
+                    unlisted_on_hp=not list_on_homepage,
+                    additional_repos=additional_repos,
+                    delete_after_days=delete_after_days,
+                )
+            except CoprRequestException as ex:
+                if "Only owners and admins may update their projects." in str(ex):
+                    logger.info(
+                        f"We have to be admins to be able to edit project settings. "
```

Comment:
Somewhat vague language. Who is "we"?

```suggestion
                        f"Admin permissions are required in order to be able to edit project settings. "
```

Reply:
`We` = user running that code. The passive does not say anything about the relevant user neither.

Reply:
Yes, that's true, but in exchange avoids the confusion of who 'we' is :slightly_smiling_face: